.pytool/Plugin/MarkdownLintCheck/MarkdownLintCheck.py

```diff
@@ -144,6 +144,7 @@ def RunBuildPlugin(self, packagename, Edk2pathObj, pkgconfig, environment, PLM,
         results = self._check_markdown(path_to_check, config_file_path, Ignores)
         for r in results:
             tc.LogStdError(r.strip())
+            logging.error(r.strip())

```

Comment:
So it looks like you have a new pattern that says if audit_only if false then report the errors to the screen at elevated level.  If audit_only then don't.  

I would be good with that pattern.  

What do you think?

Reply:
sorry...the reason these tests have an AUDIT_ONLY flag is because they are noisy and don't have 100% opt in.  So we didn't want to mess up the log with stuff that is in AUDIT_ONLY

Reply:
I've updated the markdownlintcheck to also be gated on `not audit_only`.